lib/storage/acl.js

```diff
@@ -295,13 +418,55 @@ Acl.prototype.update = function(options, callback) {
 };
 
 /**
+ * Make a {module:storage/bucket} or {module:storage/file} object private.
+ *
+ * This is a short-hand method which will remove ACL permissions for the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePrivate
+ *
+ * @example
+ * myBucket.acl.makePrivate(function(err, aclObject) {});
+ * myFile.acl.makePrivate(function(err, aclObject) {});
+ */
+Acl.prototype.makePrivate = function(callback) {
+  this.delete({
+    scope: 'allUsers'
+  }, callback);
+};
+
+/**
+ * Make a {module:storage/bucket} or {module:storage/file} object publicly
+ * readable.
+ *
+ * This is a short-hand method which will grant "reader" permissions to the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePublic
+ *
+ * @example
+ * myBucket.acl.makePublic(function(err, aclObject) {});
+ * myFile.acl.makePublic(function(err, aclObject) {});
+ */
+Acl.prototype.makePublic = function(callback) {
+  this.add({
+    scope: 'allUsers',
+    role: 'READER'
+  }, callback);
+};
```

Comment:
Basically, if it's a file, make it public. If it's a bucket, make the default acl public as well as the current files in the bucket?

I could see that being confusing since we have the `myBucket.acl.default` object as well, which would have its own `makePublic`. And in the case where a user wants to only make their current OR future files public, they would have to do the long-hand method.